cpp20_synchronization_library.bs

```diff
@@ -816,7 +813,7 @@ void atomic_flag::wait(bool old,
 <div class="numbered">
 *Effects:* Repeatedly performs the following steps, in order: 
 
-  - Evaluates `object->load(order) != old` then, if the result is `true`, returns.
+  - Evaluates `object->load(order) != old` or `this->load(order) != old` then, if the result is `true`, returns.
```

Comment:
The `this->` in function definitions that cover both free and member functions (should just be for `atomic_flag`) should stay.